simulation/sub8_gazebo/src/sub8_thruster_config.cpp

```diff
@@ -35,7 +35,7 @@ try
 catch(XmlRpcException& e)
 {
   auto err_str = "Error parsing port definitions: %s";
```

Comment:
Yup, missed that. Although err_str is interpreted by the compiler to be a char ptr.